script.js

```diff
@@ -0,0 +1,33 @@
+function calculation() {
+  let operator = document.getElementById('operator')
+  let mathOperators = operator.options[operator.selectedIndex].value
+  let answer = document.getElementById('answer')
+  let num1 = Number(document.getElementById('num1').value)
+  let num2 = Number(document.getElementById('num2').value)
+
+  function mathValidation(){
+   if ((num1 == "") || (num2 == "")){
+   alert("Input is not valid. Please enter a number in all fields.");
+   }
+  }
+
+   mathValidation()
+
+  switch (mathOperators) {
+    case 'add':
+      answer.innerHTML = num1 + num2;
+      break
```

Comment:
instead of doing answer.innerHTML each time in line 18,22,16,30.

you can store the value into a variable. then on line 33 can add  "answer.innerHTML = <variable name>"